.devcontainer/resources/phpbb-config.yml

```diff
@@ -0,0 +1,38 @@
+installer:
+    admin:
+        name: admin
+        password: mypassword
```

Comment:
Maybe just `adminadmin` similar to the default user in unit tests might make sense here.